.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This folder is still present.

Reply:
@me-x-mi I think it will be impossible to have two routes referencing the same component like here. The `About` being referenced twice.
```javascript
<Route path="/about" component={About} />

<Route exact path="/about/readme" component={About} />
```
However, your right about this.
```javascript
path="/about/readme"
```
From this [link](https://stackoverflow.com/questions/49162311/react-difference-between-route-exact-path-and-route-path) it is clear that when you use:
```javascript 
<Switch> 
```
you don't need to include `exact` keyword on all routes but rather you have to put `exact` keyword to the Route with which it's path is also included by another Route's path. For example, home path `/` is included in all paths so it needs to have `exact` keyword to differentiate it from other paths which start with `/`. The reason would be similar to `/about` path. If you want to use another routing path like `/about/readme` which includes `/about` in it then you need to use `exact` for the `/about` route.

Reply:
@me-x-mi Remove the folder.

Reply:
Good stuff.

Reply:
But, for the purpose of security and reduce chances of getting `/article/1` issues when `/article` servers the same as `/article/1`, we can just add the `exact` to all our routes.

Reply:
@me-x-mi According to the official docs that issue can't happen:
The official docs [switch](https://reacttraining.com/react-router/web/api/Switch) and [exact](https://reacttraining.com/react-router/web/api/Route/exact-bool).